lidnd/app/[username]/[campaign_slug]/encounter/[encounter_index]/reminders.tsx

```
@@ -1,23 +1,166 @@
+  });
+
+  if (!encounter) {
+    return null;
+  }
+
+  function coerceInt(value: string | undefined) {
+    if (!value) {
+      return 0;
+    }
+    const int = parseInt(value);
+    if (isNaN(int)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)